language/bytecode-verifier/transactional-tests/tests/control_flow/break_simple.mvir

```diff
@@ -3,10 +3,12 @@
 main() {
     let x: u64;
     x = 0;
-    while (true) {
-        x = move(x) + 1;
-        break;
-    }
+label while:
+    if (true) while_body else while_cont;
```

Comment:
I thought the if is going away? Or are you still updating that?

Reply:
Yeah, I just updated this in case you wanted to see the new `Terminator` struct. Pretty much all of the Move IR tests need to be updated, they all fail.

(Meta: I tried to find a way to mark this PR as "not ready for review," "draft," "author changes planned," but I don't think GitHub has that feature 😛)